examples/pytorch/ogb_lsc/MAG240M/experimental/train_unified_tensor.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import ogb
```

Comment:
ogb is not used.

Reply:
In fact this training example is an exact copy of [https://github.com/dmlc/dgl/blob/master/examples/pytorch/ogb_lsc/MAG240M/train.py](https://github.com/dmlc/dgl/blob/master/examples/pytorch/ogb_lsc/MAG240M/train.py). Based on your comments, it seems like several improvements can be made in the baseline DGL code. However, in terms of consistency, it is kind of weird to make the updates only for this pull request. So I simply created a new example based on commonly used GraphSAGE code ([https://github.com/dmlc/dgl/blob/master/examples/pytorch/graphsage/train_sampling.py](https://github.com/dmlc/dgl/blob/master/examples/pytorch/graphsage/train_sampling.py)). Someone else can update the baseline MAG240M DGL code later.

Reply:
Yeah. In this PR we only focus on this example.
We can update the baseline coder later in another PR.